### PR TITLE
update to serilog 6

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

**Comment by SimonCropp** [comment]
@mikeminutillo thanks. still looks broken, can u post the build log here

**Comment by mikeminutillo** [comment]
@SimonCropp I figured it was easier to just fix it than to paste build logs. I'll keep an eye on it

**Comment by SimonCropp** [comment]
yep happy for u to tweak it :)

File: nservicebus/logging/serilog.md [hunk 1]
```diff
@@ -54,9 +54,80 @@ snippet: SerilogTracingLogger
...
+
+
+#### Outgoing message enrichment
+
+When a send the same properties as described in "Incoming message enrichment" will be added to the outgoing pipeline. Note that if a handler sends a message, the logger injected into the outgoing pipeline will be forked from the logger instance as described in "Handler enrichment". As such it will contain a property `Handler` for the handler that sent the message.
+
+
+#### Accessing the logger
+
+The contextual logger instance can be access from anywhere in the pipeline via `SerilogTracingExtensions.Logger(this IPipelineContext context)`.
```
Comment (mikeminutillo) [inline nservicebus/logging/serilog.md]:
```suggestion
The contextual logger instance can be accessed from anywhere in the pipeline via `SerilogTracingExtensions.Logger(this IPipelineContext context)`.
```

**Comment by SimonCropp** [comment]
think i have address all the feetback. thanks for the review

**Comment by mikeminutillo** [comment]
Thanks for the update :)

**Pull request merged**